Makefile

```diff
@@ -45,6 +45,9 @@ update-local-config:
 	@echo "Updating local config..."
 	@go generate ${CONFIG_DIR}
 
+diff-local-configs: update-local-config
```

Comment:
This is going to fail everytime because it runs `update-local-config`.  The output of this is nondeterministic due to map iteration

Reply:
Yes I realized that after running it a few times :(
Maybe I can sort before serialization